apps/server/src/app.ts

```diff
@@ -69,6 +72,23 @@ app.use(function (err: HttpError, req: Request, res: Response) {
   res.render("error");
 });
 
+// Define routes for registering as a customer or a rider
+app.post('/register/customer', async (req, res) => {
+  await registerCustomer(req, res);
+});
+
+app.post('/register/rider', async (req, res) => {
+  await registerRider(req, res);
+});
+
+// Define route for retrieving rider details
+app.get('/riders', async (req, res) => {
+  await getRiderDetails(req, res);
+});
+app.get('/rider:id', async (req, res) => {
+  await getRiderById(req, res);
```

Comment:
Why do you need to async await the function when routing when you've already done that in the function? this is wrong. Remove the async await, please.